pkg/lvm/mount.go

```diff
@@ -198,7 +198,12 @@ func MountVolume(vol *apis.LVMVolume, mount *MountInfo, podLVInfo *PodLVInfo) er
 
 	err = FormatAndMountVol(devicePath, mount)
 	if err != nil {
-		return status.Error(codes.Internal, "not able to format and mount the volume")
+		return status.Error(
+			codes.Internal,
+			fmt.Sprintf(
```

Comment:
you can use status.Errorf with %s format, fmt is not needed.

Reply:
Updated with Errorf